SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Shared/Crashes.cs

```diff
@@ -7,172 +7,179 @@
+		/// </summary>
+		/// <value>
+		/// Task with result being <c>true</c> if a crash was recorded in the last session, otherwise <c>false</c>.
+		/// </value>
 #endif
-        public static Task<bool> HasCrashedInLastSessionAsync() => PlatformHasCrashedInLastSessionAsync();
+		public static Task<bool> HasCrashedInLastSessionAsync() => PlatformHasCrashedInLastSessionAsync();
 
 #if USES_WATSON
         [EditorBrowsable(EditorBrowsableState.Never)]
         [Obsolete("This does not exist in UWP and should not be used.")]
 #else
-        /// <summary>
-        /// Gets the crash report generated in the last session if there was a crash.
-        /// </summary>
-        /// <value>Crash report from the last session, <c>null</c> if there was no crash in the last session.</value>
+		/// <summary>
+		/// Gets the crash report generated in the last session if there was a crash.
+		/// </summary>
+		/// <value>Crash report from the last session, <c>null</c> if there was no crash in the last session.</value>
 #endif
-        public static Task<ErrorReport> GetLastSessionCrashReportAsync()
-        {
-            return PlatformGetLastSessionCrashReportAsync();
-        }
+		public static Task<ErrorReport> GetLastSessionCrashReportAsync()
+		{
+			return PlatformGetLastSessionCrashReportAsync();
+		}
 
-        /// <summary>
-        /// Generates crash for testing purposes.
-        /// </summary>
-        /// <remarks>
-        /// This call has no effect in non debug configuration (such as release).
-        /// </remarks>
-        [System.Diagnostics.Conditional("DEBUG")]
-        public static void GenerateTestCrash()
-        {
-            throw new TestCrashException();
-        }
+		/// <summary>
+		/// Generates crash for testing purposes.
+		/// </summary>
+		/// <remarks>
+		/// This call has no effect in non debug configuration (such as release).
+		/// </remarks>
+		[System.Diagnostics.Conditional("DEBUG")]
+		public static void GenerateTestCrash()
+		{
+			throw new TestCrashException();
+		}
 
 #if USES_WATSON
         [EditorBrowsable(EditorBrowsableState.Never)]
         [Obsolete("This does not exist in UWP and should not be used.")]
 #else
-        /// <summary>
-        /// Track a handled error.
-        /// </summary>
-        /// <param name="exception">The .NET exception describing the handled error.</param>
-        /// <param name="properties">Optional properties.</param>
+		/// <summary>
+		/// Track a handled error.
+		/// </summary>
+		/// <param name="exception">The .NET exception describing the handled error.</param>
+		/// <param name="properties">Optional properties.</param>
 #endif
-        public static void TrackError(Exception exception, IDictionary<string, string> properties = null)
-        {
-            PlatformTrackError(exception, properties);
-        }
-    }
+		public static void TrackError(Exception exception, IDictionary<string, string> properties = null)
+		{
+			if (exception == null)
```

Comment:
File was formatted incorrectly :/
This is the method that was actually patched with the if and log message.